plugins/woocommerce/changelog/61963-task-update-auth-capture-logic

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
+Make capture later flow more secure to ensure the usage of correct authorization ID in PayPal Standard.
```

Comment:
Nit: I think it would be better to include PayPal Standard much earlier in the description, as that's a fairly critical component of this change when considering that "capture later" could apply to many other flows in WooCommerce. I am also not sure how this makes things "more secure".
```suggestion
Ensure PayPal Standard capture later flow uses the correct authorization ID.
```